dissect/target/plugins/os/windows/defender_helpers/defender_records.py

```diff
@@ -0,0 +1,176 @@
+from dissect.target.helpers.record import TargetRecordDescriptor
+
+DefenderMPLogProcessImageRecord = TargetRecordDescriptor(
```

Comment:
I tried to follow the attached example program as closely as possible. If you find my solution deviating please tell me how. Otherwise, if you like more data, please extend the issue with your wishes. Since I am not a domain expert myself, it's hard for me to judge.

Reply:
Oh, it was my observation that there is usually just 1 file, so I thought it would cause a lot of overhead. But I can add it anyway.